lesson03/assignment/css/_body.components.scss

```diff
@@ -0,0 +1,15 @@
+body {
+  background-color: $primary-bg-color;
+}
+
+.main {
+  max-width: 750px;
```

Comment:
Core to responsive design is avoiding static units of measurement. Almost always, we should be using rem or %.